src/components/Platform.jsx

```diff
@@ -46,14 +43,13 @@ class Platform extends Component {
   render() {
     return (
       <div>
-        <h1 className="content-subhead">{'Candidates for ' + this.props.platform.candidacy}</h1>
+        <h1 className="content-subhead">{"Candidates for " + this.props.platform.candidacy}</h1>
 
         <section className="post">
           <header className="post-header">
 
             <h2 className="post-title">{this.props.platform.name}</h2>
-
-            <Picture name={this.props.platform.name} image={this.props.platform.image}/>
+            {Picture(this.props.platform.name, this.props.platform.image)}
```

Comment:
shouldn't do this: should keep it like 
` <Picture name={this.props.platform.name} image={this.props.platform.image}/>`